src/Library/demos/Boxed lists/main.js

```diff
@@ -0,0 +1,11 @@
+const button = workbench.builder.get_object("button");
```

Comment:
Apparently this expression is throwing errors in the terminal. I'm not well experienced with gjs, so I have no idea how to fix this.

Reply:
Nvm I think I fixed it. It was related to the previous issue you pointed out https://github.com/sonnyp/Workbench/pull/210#discussion_r1148580938